### PR TITLE
feat: update welcome message for A380X

### PR DESCRIPTION
**Pull request by ExampleWasTaken**
## Description
Now that the A380X has been released, I think the server welcome-message should reflect that.

I pretty much just exchanged "A32NX" for "aircraft" but marketing may want to have a quick check. :)

Also, there is no `#a380x-support` channel on the staging server so the staging config, currently has no channel set for that.

## Test Results
Note that emojis are hardcoded to the FBW server so they don't show on my testing server.

image
image
image

## Discord Username
examplewastaken

### CODE REVIEW

**Comment by benw202** [comment]
@Valastiri - Can you check how this reads? Thought I would loop you in to give this a check from a marketing point of view

File: src/commands/moderation/welcome.ts [hunk 1]
```diff
@@ -19,9 +19,7 @@ const WELCOME_EMBED = makeEmbed({
     description: makeLines([
         'Welcome to the **Official Discord Server** of **FlyByWire Simulations!**',
         '',
-        'The A32NX Project is a community-driven open source project to create a free Airbus A320neo in Microsoft Flight Simulator that is as close to reality as possible. It started out as an enhancement project to the default A320neo and is now proceeding as an independent add-on project aiming to bring the FlyByWire A32NX up to payware-level systems depth and functionality, all for free.',
-        '',
-        'We are also developing an A380 from scratch which will be aiming to produce a high fidelity freeware aircraft.',
+        'The FlyByWire Simulations is a community-driven open source project to create a free Airbus A320neo and Airbus A380 in Microsoft Flight Simulator that are as close to reality as possible. It started out as an enhancement project to the default A320neo and is now proceeding as an independent add-on project aiming to bring both aircraft up to payware-level systems depth and functionality, all for free.',
```
Comment (benw202) [inline src/commands/moderation/welcome.ts]:
```suggestion
        'FlyByWire Simulations is a community-driven, open-source project aimed at creating realistic Airbus A320neo and Airbus A380 aircraft for Microsoft Flight Simulator. Originally launched as an enhancement project for the default A320neo, it has evolved into an independent add-on project that strives to bring both aircraft to a payware-level standard in terms of systems depth and functionality - all for free',
```

Rewrite with some grammar changes

File: src/commands/moderation/welcome.ts [hunk 2]
```diff
@@ -19,9 +19,7 @@ const WELCOME_EMBED = makeEmbed({
...
         '',
         `Feel free to download, test, and share your feedback, or if you are interested in developing, assign your <#${constantsConfig.channels.ROLES}>, and get cracking!`,
```
Comment (benw202) [inline src/commands/moderation/welcome.ts]:
```suggestion
        `Feel free to download, test, and share your feedback. If you are interested in developing, assign your <#${constantsConfig.channels.ROLES}>, and get cracking!`,
```

**Review by benw202 — COMMENTED** [review]
Just a couple of ideas. Feel free to reject!

**Comment by ExampleWasTaken** [comment]
> Just a couple of ideas. Feel free to reject!

Awesome suggestions, thank you!

**Review by pdellaert — APPROVED** [review]

**Review by benw202 — APPROVED** [review]

**Pull request merged**